mmv1/third_party/terraform/services/compute/resource_compute_router_peer.go.erb

```diff
@@ -32,6 +32,15 @@ func ipv6RepresentationDiffSuppress(_, old, new string, d *schema.ResourceData)
 	return oldIp.Equal(newIp)
 }
 
+func isIPv4Address(input string) bool {
+	ip := net.ParseIP(input)
+	return ip != nil && ip.To4() != nil
+}
+
+func interfaceToString(value interface{}) string {
+	return fmt.Sprintf("%v", value)
+}
```

Comment:
It look like these two function are not used anywhere?

Reply:
Removed